identity/test/pool.go

```diff
@@ -88,7 +88,6 @@ func TestPool(ctx context.Context, conf *config.Config, p persistence.Persister,
 		})
 
 		t.Run("case=expand", func(t *testing.T) {
```

Comment:
Could you add an explicit pagination test here for both approaches?

Reply:
Added an assertion, please confirm that matches what you expected 👍